upp/configs/extended_labels.yaml

```diff
@@ -0,0 +1,92 @@
+variables: !include variables.yaml
+global_cuts: !include splits/simple-split.yaml
+
+# for k-folded splits, only required for final trainings of GN2
+# global_cuts: !include splits/fold0.yaml
+
+ttbar: &ttbar
+  name: ttbar
+  equal_jets: False
+  pattern:
+    - "user.svanstro.410470.e6337_s3681_r13144_p5922.tdd.EMPFlow_kfold.24_2_27.23-11-10_kfoldtrain_output.h5/*.h5"
```

Comment:
Just for my information - so far your studies only trained on ttbar?

Reply:
Yeah, uptil now all the trainings for both default and extended are only trained on ttbar

Reply:
Good to know, thanks!